src/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
```

Comment:
The variable `$gap` is used here but not defined within this file. Ensure it is defined elsewhere in your project or imported if defined in another file.